css/index.css

```diff
@@ -109,182 +109,207 @@ margin-bottom: 20px;
-    }
-    nav {
-        margin-left: 8.4%;
-        margin-top: 20px;
-    }
-
-    .lambda-logo {
-        margin-left: 38.5%;
-        margin-top: 22px;
-    }
-    .text-container h2 {
-        font-size: 1rem;
-    }
-
-    .text-container h2 p {
-        font-size: 14px;
-    }
-    .middle-content h2{
-        font-size: 1rem;
-    }
-
-    .top-content .text-container p {
-        font-size: 14px;
-    }
-    .middle-content .boxes .box {
-        height: 84px;
-    }
-
-    .middle-content .boxes .box1 {
-        background: teal;
-    }
-    .middle-content .boxes .box3 {
-        background: gold;
-    }
-    .middle-content .boxes .box5 {
-        background: cadetblue;
-    }
-    .middle-content .boxes .box7 {
-        background: coral;
-    }
-    .middle-content .boxes .box9 {
-        background: crimson;
-    }
-
-    .bottom-content .text-container{
-        font-size: 14px;
-    }
-    footer nav {
-        padding: 18px 0%;
-        font-size: 1.2rem;
-        height: 43px;
-    }
+header {
+display: block;
+}
+nav {
+margin-left: 8.4%;
+margin-top: 20px;
+}
+
+.lambda-logo {
+margin-left: 38.5%;
+margin-top: 22px;
+}
+.text-container h2 {
+font-size: 1rem;
+}
+
+.text-container h2 p {
+font-size: 14px;
+}
+.middle-content h2{
+font-size: 1rem;
+}
+
+.top-content .text-container p {
+font-size: 14px;
+}
+.middle-content .boxes .box {
+height: 84px;
+}
+
+.middle-content .boxes .box1 {
+background: teal;
+}
+.middle-content .boxes .box3 {
+background: gold;
+}
+.middle-content .boxes .box5 {
+background: cadetblue;
+}
+.middle-content .boxes .box7 {
+background: coral;
+}
+.middle-content .boxes .box9 {
+background: crimson;
+}
+
+.bottom-content .text-container{
+font-size: 14px;
+}
+footer nav {
+padding: 18px 0%;
+font-size: 1.2rem;
+height: 43px;
+}
 }
 
 
 /* Iphone */
 
 @media (max-width: 400px) {
-    .container {
-        width: 100%;
-    }
-    header {
-        flex-wrap: wrap;
-        justify-content: center;
-    }
-    header nav {
-        width: 100%;
-    }
-
-    .top-content {
-        display: block;
-    }
-    .top-content .text-container {
-        width: 98%;
-        margin-left: 2.5%;
-    }
-    .top-content .text-container p {
-        font-size: 13px;
-    }
-    .middle-content .boxes .box10 {
-        display: none;
-    }
-    .bottom-content {
-        flex-direction: column;
-    }
-    .bottom-content .text-container {
-        margin-bottom: 3%;
-    }
-    footer nav {
-        font-size: 0.4rem;
-        margin-left: 78px;
-    }
+.container {
+width: 100%;
+}
+header {
+display: flex;
+flex-wrap: wrap;
+justify-content: center;
+}
+header nav {
+width: 100%;
+display: flex;
+flex-direction: column;
+align-items: center;
+}
+header nav .a1 {
+    padding: 26px;
+}
+header nav .a2 {
+    padding-bottom: 31px;
+}
+header nav .a3 {
+    padding-bottom: 29px;
+}
+header nav .a4 {
+    padding-bottom: 29px;
+}
+
+
+header .lambda-logo {
+    margin-left: 8.5%;
+}
+.top-content {
```

Comment:
Here I would do this instead:

```
.top-content {
  flex-direction: column;
  align-items: center;
  width: 100%;
}

.top-content .text-container {
   width: 90%;
}
```

That way your `.text-container` content sits nicely centered in the `.top-content` `<div>` with some breathing room on both sides.